playbooks/roles/edx_ansible/templates/silent_json_update.j2

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [[ -f {{ edx_ansible_var_file }} ]]; then
+    extra_args="-e@{{ edx_ansible_var_file }}"
+fi
```

Comment:
else
    echo "file /edx/app/edx_ansible/server-vars.yml not found. exiting..." >&2
    exit 1
fi

for cfile in ~edxapp/*.json ; do
    cp -fap ${cfile} ${cfile}.bak || exit 1
done